docs/releasing.md

```diff
@@ -47,75 +32,63 @@ IPAM uses [semantic versioning](https://semver.org).
-  If the release tag build is not visible, check if the action has failed and
-  retrigger as necessary.
+- Switch to the main branch: `git checkout main`
 
-We also need to create one or more tags for the Go modules ecosystem:
+- Create a new branch for the release notes**:
+  `git checkout -b release-notes-1.x.x origin/main`
 
-- For any subdirectory with `go.mod` in it (excluding `hack/tools`), create
-  another Git tag with directory prefix, ie.
-  `git tag api/v1.x.y`.
+- Generate the release notes: `RELEASE_TAG=v1.x.x make release-notes`
+   - Replace `v1.x.x` with the new release tag you're creating.
+   - This command generates the release notes here
+     `releasenotes/<RELEASE_TAG>.md` .
 
-  **NOTE**: Do not create annotated tags (`-a`, or implicitly via `-m` or `-s`)
-  for Go modules. Release notes expects only the main tag to be annotated,
-  otherwise it might create incorrect release notes.
-  Push the tag to `origin`.
-
-### Release notes
-
-Next step is to clean up the release note manually. Release note has been
-generated by the `release` action, do not click the `Generate release notes`
-button. In case there is issue with release action, you may rerun it via
-`Actions` tab, or you can `make release-notes` to get a markdown file with
-the release content to be inserted.
-
-- If release is not a beta or release candidate, check for duplicates, reverts,
-  and incorrect classifications of PRs, and whatever release creation tagged to
-  be manually checked.
+- Next step is to clean up the release note manually.
+   - If release is not a beta or release candidate, check for duplicates,
+     reverts, and incorrect classifications of PRs, and whatever release
+     creation tagged to be manually checked.
    - For any superseded PRs (like same dependency uplifted multiple times, or
-     commit revertions) that provide no value to the release, move them to
+     commit reversions) that provide no value to the release, move them to
      Superseded section. This way the changes are acknowledged to be part of the
      release, but not overwhelming the important changes contained by the
      release.
-- If the release you're making is not a new major release, new minor release,
-  or a new patch release from the latest release branch, uncheck the box for
-  latest release.
-- If it is a release candidate (RC) or a beta release, tick pre-release box.
-- Save the release note as a draft, and have others review it.
+
+- Commit and push your changes, push the new branch and create a pull request.
+IMPORTANT_NOTE:
+   - The commit and PR title should be 🚀 Release v1.x.y.
+   - Important! The commit should only contain the release notes file, nothing
+     else, otherwise automation will not work.
+
+- Ask maintainers and release team members to review your pull request.
+
+Once PR is merged it triggers three things:
+
+- Github action `push_release_tags` will create and push the tags. This action
```

Comment:
There is no action with this name. All were adding is `release.yml` with `Create release` name.

What you're referring are jobs in the workflow.

Reply:
fixed.